modular_skyrat/modules/customization/modules/surgery/organs/synth.dm

```diff
@@ -210,29 +207,29 @@
 			to_chat(user, span_warning("[target_apc]'s emergency power is active."))
 			break
 		// Calculate how much to draw this cycle
-		var/power_use = clamp(power_needed, IPC_CHARGE_MIN, IPC_CHARGE_MAX)
+		var/power_use = clamp(power_needed, SYNTH_CHARGE_MIN, SYNTH_CHARGE_MAX)
 		power_use = clamp(power_use, 0, target_apc.cell.charge)
 		// Are we able to draw anything?
 		if(power_use==0)
 			to_chat(user, span_warning("[target_apc] lacks the power to charge you."))
 			break
 		// Calculate the delay.
-		var/power_delay = (power_use/100) * IPC_CHARGE_DELAY_PER_100
+		var/power_delay = (power_use/100) * SYNTH_CHARGE_DELAY_PER_100
```

Comment:
I still don't totally get this /100 but not really your fault I guess.
```suggestion
		var/power_delay = (power_use / 100) * SYNTH_CHARGE_DELAY_PER_100
```

Reply:
I plan on overhauling a whole bunch of this code in part 2, don't worry.

Reply:
Cause ew ew ewewewewewew `while(TRUE)`